src/test/ui/parser/removed-syntax-extern-const-fn.rs

```diff
@@ -0,0 +1,8 @@
+extern {
```

Comment:
The filename could be improved a bit: `no-const-fn-in-extern-block.rs` (since no syntax has been removed, merely not added).